Moovies/Moovies/Repository.swift

```diff
@@ -0,0 +1,150 @@
+// Repository.swift
+// Copyright © RoadMap. All rights reserved.
+
+import CoreData
+import Foundation
+
+protocol DatabaseProtocol {
+    func get(movieType: MoviesType) -> [Movie]
+    func getDescription(id: Int) -> [Description]
+    func add(object: [Movie], movieType: MoviesType)
+    func addDescription(object: [Description], id: Int)
+    func remove(id: Int)
+    func removeAll()
+}
+
+final class CoreDataMovies: DatabaseProtocol {
+    func getDescription(id: Int) -> [Description] {
+        return []
+    }
+
+    func addDescription(object: [Description], id: Int) {}
+
+    let coreDataService = CoreDataService.shared
```

Comment:
```suggestion
  private  let coreDataService = CoreDataService.shared
```
свойство выше методов